build.gradle

```diff
@@ -0,0 +1,175 @@
+import org.apache.tools.ant.filters.ReplaceTokens
+
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+    id 'checkstyle'
+    id 'com.github.johnrengelman.shadow' version '7.1.2'
+    id 'org.jetbrains.intellij' version '1.1'
+}
+
+version = '4.3.2-SNAPSHOT'
+group = 'com.onarandombox.multiversecore'
+description = 'Multiverse-Core'
+
+java.sourceCompatibility = JavaVersion.VERSION_11
+
+repositories {
+    mavenLocal()
+    maven {
+        url = uri('https://hub.spigotmc.org/nexus/content/repositories/snapshots/')
+    }
+
+    maven {
+        url = uri('https://repo.onarandombox.com/content/groups/public')
+    }
+
+    maven {
+        url = uri('https://hub.spigotmc.org/nexus/content/groups/public/')
+    }
+
+    maven {
+        url = uri('https://jitpack.io')
+    }
+
+    maven {
+        url = uri('https://repo.minebench.de/')
+    }
+
+    maven {
+        url = uri('https://repo.maven.apache.org/maven2/')
+    }
+}
+
+dependencies {
+    implementation('org.bukkit:bukkit:1.13.2-R0.1-SNAPSHOT') {
+        exclude group: 'junit', module: 'junit'
+    }
+
+    implementation('com.github.MilkBowl:VaultAPI:1.7') {
+        exclude group: 'org.bukkit', module: 'bukkit'
+    }
+
+    api('me.main__.util:SerializationConfig:1.7') {
+        exclude group: 'org.bukkit', module: 'bukkit'
+    }
+    api('com.pneumaticraft.commandhandler:CommandHandler:11') {
+        exclude group: 'org.bukkit', module: 'bukkit'
+        exclude group: 'junit', module: 'junit'
+    }
+    api 'com.dumptruckman.minecraft:buscript:2.0-SNAPSHOT'
```

Comment:
May want to remove this as Buscript has been removed now in the MV5 branch

Reply:
This is targeting main, so it should be kept. I will change when I merge MV5